pkg/registry/client.go

```diff
@@ -113,7 +113,7 @@ func (c client) Write(ref name.Reference, image v1.Image) error {
 func (c client) GetDigest(image string) (string, error) {
 	descriptor, err := c.GetDescriptor(image)
 	if err != nil {
-		return "", fmt.Errorf("error getting image digest: %w", err)
```

Comment:
This is redundant because the calling function already wraps this error making the message look like the following:

`Command execution failed: error getting image metadata: error getting image tag with digest: error getting image digest: error getting image descriptor: get https://gcr.io/XXX`

With this change:

`Command execution failed: error getting image metadata: error getting image tag with digest: error getting image descriptor: get`